client/signup/jetpack-connect/index.jsx

```diff
@@ -17,6 +17,13 @@ import Main from 'components/main';
 import JetpackConnectNotices from './jetpack-connect-notices';
 import SiteURLInput from './site-url-input';
 import { dismissUrl, goToRemoteAuth, goToPluginInstall, checkUrl } from 'state/jetpack-connect/actions';
+import versionCompare from 'lib/version-compare';
+
+
+/**
+ * Constants
+ */
+const MINIMUM_JETPACK_VERSION = '3.9.6';
 
```

Comment:
Yeah, it should get bumped. CC @eliorivero - make sure to bump version number:
https://github.com/Automattic/jetpack/blob/master/readme.txt#L4